plugins/experimental/wasm/ats_context.cc

```diff
@@ -565,11 +565,17 @@ Context::httpCall(std::string_view target, const Pairs &request_headers, std::st
       path = value;
     } else if (key == ":authority") {
       authority = value;
+    } else if (key == ":protocol") {
```

Comment:
It should be ":scheme" instead according to 
https://portswigger.net/burp/documentation/desktop/http2/http2-basics-for-burp-users
Please check out the section in pseudo-headers.